src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -197,6 +199,24 @@ ExtensionBooleanResponse handleIndicesModuleNameRequest(IndicesModuleRequest ind
         return indicesModuleNameResponse;
     }
 
+    /**
+     * Handles a request from OpenSearch to execute a REST request on the extension.
+     *
+     * @param request The REST request to execute
+     * @return A response acknowledging the request.
+     */
+    RestExecuteOnExtensionResponse handleRestExecuteOnExtensionRequest(RestExecuteOnExtensionRequest request) {
+        String message = "The extension would have just executed " + request.getMethod() + " " + request.getUri();
+        // TODO: logic matching the method/URI which came from the extension API to the appropriate
+        // action class, e.g., see AD plugin's actions in org.opensearch.ad.rest package.
+        // This should probably be stored locally in a map before we send the register API requests to OpenSearch.
+        // Tricky part is how to match up API names (in text) with actions (Class Names). Could use reflection
+        // or just register like the existing plugin code does. TBD future code!
+        // For now we just log (locally) and respond to OpenSearch that we received enough info to correlate such an action
```

Comment:
Can we open an issue for this TODO which will basically help the extension to redirect the request to the appropriate actions?

Reply:
#102